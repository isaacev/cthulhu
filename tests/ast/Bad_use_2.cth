use ::Foo::Bar*;

fn foo() -> () {
  -- do stuff
}
