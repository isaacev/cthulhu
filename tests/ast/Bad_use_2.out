
  EXPECTED TOKEN  ------------------------------------------------------------

    : Bad_use_2
  1 | use ::Foo::Bar*;
    |               ^ expected ; here

